vcbuild.bat

```diff
@@ -313,7 +321,9 @@ if "%target%"=="Build" (
   if defined cctest set target="Build"
 )
 if "%target%"=="rename_node_bin_win" if exist "%config%\cctest.exe" del "%config%\cctest.exe"
-msbuild node.sln %msbcpu% /t:%target% /p:Configuration=%config% /p:Platform=%msbplatform% /clp:NoSummary;NoItemAndPropertyList;Verbosity=minimal /nologo
+set binary_logging_flag=^ 
+if defined binlog set binary_logging_flag=/binaryLogger:%config%\node.binlog
+msbuild node.sln %msbcpu% /t:%target% /p:Configuration=%config% /p:Platform=%msbplatform% /clp:NoSummary;NoItemAndPropertyList;Verbosity=minimal %binary_logging_flag% /nologo
```

Comment:
```suggestion
msbuild node.sln %msbcpu% /t:%target% /p:Configuration=%config% /p:Platform=%msbplatform% /clp:NoSummary;NoItemAndPropertyList;Verbosity=minimal /nologo %extra_msbuild_args%
```

I think `%extra_msbuild_args%` is a more optimal solution...
1. It allows free naming and path of a binlog
2. It allows setting verbosity level of the another logger (with -flp)
3. It allows other custom args

Also the binlog use case is non-trivial as it requires a tool to read it.

Reply:
Understood. Would you be OK with binlog as a special case of `extra_msbuild_args%?

Something like
```cmd
if defined binlog (
    set extra_msbuild_args=%extra_msbuild_args% /binaryLogger:%config%\node.binlog
)
```
combined with a generic way to pass in args to `extra_msbuild_args`? (I'll start prototyping this now)

Reply:
Sure, makes sense.

Reply:
We might even add this to CI, and export the binlog as an artifact...

Reply:
Now that I've coded it up, the `extra_msbuild_args` path feels much cleaner. Thank you.

As for exporting it as an artifact, if you're willing to use the extra tool then it's easy to find the complete command line for any arbitrary step--especially failed ones. Not a bad idea :)

Reply:
I've actually stopped using `vcbuild.bat` because of the lack of a custom arg espace hatch.
One arg I like to use is `/p:BuildProjectReferences=false` so I run msbuild directly...

Reply:
Makes sense to me, @refack. I was doing the same for a while until I started touching project generation. :)

What's the next step here?